### PR TITLE
fix(suite-native): polished discreet mode

### LINKED ISSUE #8679: Polish the discreet mode appearance
The discreet mode does not look great. Make some UI improvements to make it look better.

### PR DESCRIPTION
**Pull request by PeKne**
DiscreetText appearance was improved to look better.

Closes #8679

## Screenshots:

https://github.com/trezor/trezor-suite/assets/26143964/6f4c36bb-4c15-473a-a39c-26090e5be1f1

### CODE REVIEW

**Review by juriczech — APPROVED** [review]

File: suite-native/atoms/src/DiscreetText.tsx [hunk 1]
```diff
@@ -16,9 +16,9 @@ type DiscreetTextProps = TextProps & {
 };
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Regular.otf');
+const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Medium.otf');
 
-const isDiscreetModeOn = atomWithUnecryptedStorage<boolean>('isDiscreetModeOn', false);
+const isDiscreetModeOn = atomWithUnecryptedStorage<boolean>('isDiscreetModeOn', true);
```
Comment (Nodonisko) [inline suite-native/atoms/src/DiscreetText.tsx]: This should be false by default?

File: suite-native/atoms/src/DiscreetText.tsx [hunk 2]
```diff
@@ -121,7 +128,7 @@ export const DiscreetText = ({
                 variant={variant}
                 color={color}
                 onLayout={handleLayout}
-                style={applyStyle(textStyle, { isDiscreetMode })}
+                style={applyStyle(textTemplateStyle, { isDiscreetMode })}
```
Comment (Nodonisko) [inline suite-native/atoms/src/DiscreetText.tsx]: Variable isDiscreetMode seems unused inside of style

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: suite-native/atoms/src/DiscreetText.tsx (hunk 1 above)
Comment (PeKne) [reply suite-native/atoms/src/DiscreetText.tsx]: You are right. I had changed it for debugging and then forgot to change it back. fixed in 94ef45ffb.

File: suite-native/atoms/src/DiscreetText.tsx (hunk 2 above)
Comment (PeKne) [reply suite-native/atoms/src/DiscreetText.tsx]: fixed in 94ef45ffb

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**